tests/Query/BuilderTest.php

```diff
@@ -1600,12 +1599,38 @@ public static function getEloquentMethodsNotSupported()
         yield 'orWhereIntegerNotInRaw' => [fn (Builder $builder) => $builder->orWhereIntegerNotInRaw('id', ['1a', 2])];
     }
 
-    private static function getBuilder(): Builder
+    public function testRenameEmbeddedIdFieldCanBeDisabled()
```

Comment:
Does this test exercise both `aliasIdForQuery` _and_ `aliasIdForResult()`?

Reply:
Thanks for this question, I added a full test to cover both.